backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -31,31 +40,86 @@ object BackendWorkflowInitializationActor {
+    case Initialize => performActionThenRespond(initSequence(), onFailure = InitializationFailed)
+    case AbortWorkflow => performActionThenRespond(abortInitialization(), onFailure = BackendWorkflowAbortFailedResponse)
   }
 
   /**
     * Our predefined sequence to run during preStart
     */
-  final def initSequence = for {
-    _ <- validate
-    _ <- beforeAll
-  } yield InitializationSuccess
+  final def initSequence() = {
+    for {
+      _ <- validate()
+      _ <- beforeAll()
+    } yield InitializationSuccess
+  }
 
   /**
     * Abort all initializations.
     */
-  def abortInitialization: Future[WorkflowAbortResponse]
+  def abortInitialization(): Future[WorkflowAbortResponse]
+
+  /**
+    * A call which happens before anything else runs
+    */
+  def beforeAll(): Future[Unit]
+
+  /**
+    * Validates runtime attributes for one specific call.
+    *
+    * @param runtimeAttributes Runtime Attributes with already evaluated values.
+    * @return If all entries from runtime attributes section are valid Success otherwise
+    *         Failure with the aggregation of errors.
+    */
+  def validateRuntimeAttributes(runtimeAttributes: EvaluatedRuntimeAttributes): ErrorOr[Product]
 
   /**
     * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
     */
-  def validate: Future[Unit]
+  private def validate(): Future[Unit] = {
+    Future {
+      val callsWithValidations = calls map { call =>
+        call -> validateRuntimeAttributes(evaluateRuntimeAttributesWdlExpressions(call))
+      }
+      val failedValidations = callsWithValidations collect {
+        case (call, validation) if validation.isFailure => {
+          validation match {
+            case failureWithStringErr: Failure[NonEmptyList[String]] =>
+              ValidationAggregatedException(s"Runtime attribute validation failed for task '${call.taskFqn}'.", failureWithStringErr.e.list)
+            case Success(_) => new IllegalStateException("Scalaz validation says it's a failure but type is Success. It should not happen.")
+          }
+        }
+      }
+
+      if (failedValidations.nonEmpty) {
+        throw new AggregatedException("Runtime attribute validation failed", failedValidations)
+      }
+    }
+  }
 
   /**
-    * A call which happens before anything else runs
+    * Evaluates WDL expressions within call runtime attributes.
+    *
+    * @param call Call which contains WDL expressions in it.
+    * @return Evaluated runtime attributes.
     */
-  def beforeAll: Future[Unit]
+  private def evaluateRuntimeAttributesWdlExpressions(call: Call): EvaluatedRuntimeAttributes = {
+    val evaluator = createEvaluator(call)
+    def evaluate(wdlExpression: WdlExpression) = evaluator.evaluate(wdlExpression.ast)
+    val evaluateAttrs = call.task.runtimeAttributes.attrs mapValues evaluate
+    TryUtils.sequenceMap(evaluateAttrs, "Runtime attributes evaluation").get
```

Comment:
What happens if this `.get` throws an exception?

It might be that we just couldn't evaluate something yet because we haven't started running the workflow. That doesn't mean validation should fail.